posthog/models/event.py

```diff
@@ -92,14 +90,9 @@ def _element_subquery(
         subqueries = {}
         for index, tag in enumerate(selector.parts):
             subqueries["match_{}".format(index)] = Subquery(
-                Element.objects.filter(
-                    group_id=OuterRef("pk"),
-                    **tag.data
-                ).values(
+                Element.objects.filter(group_id=OuterRef("pk"), **tag.data).values(
                     "order"
-                ).order_by('order')
```

Comment:
almost 100% sure this is the culprit but want to just revert for now before I attempt a fix